desktop/apps/consignments2/routes.js

```diff
@@ -2,7 +2,9 @@ import Items from '../../collections/items'
 import JSONPage from '../../components/json_page'
 import markdown from '../../components/util/markdown'
 import metaphysics from '../../../lib/metaphysics'
```

Comment:
This is a nice time to take care of our new absolute path resolution 😄  `import metaphysics from 'lib/metaphysics.coffee'`